test/unit/volume_manager_btrfs_test.py

```diff
@@ -1,7 +1,10 @@
-
 from mock import patch
 from mock import call
 import mock
+import os
+
+from textwrap import dedent
+import datetime
 
 from .test_helper import *
```

Comment:
Let me [save 128K of words](https://pythonconquerstheuniverse.wordpress.com/2011/03/28/why-import-star-is-a-bad-idea/) for this. :smile: